doc/contributing/reconizing-contributors.md

```diff
@@ -23,7 +23,8 @@ Some of the benefits we hope to achieve through these programs include:
 ## Bi-monthly contributor spotlight
 
 The contributor spotlight program showcases individual(s) or teams who have
-made signification contributions to the project. Anybody who has made a
+made signification contributions to the project. Anybody who is not
+a voting or regular member of the TSC and has made a
```

Comment:
Aren't all members of the TSC either voting or regular members? (i.e. we don't need to call out "voting or regular" and just say "a member of the TSC".
```suggestion
made signification contributions to the project. Anybody who is not
a member of the Technical Steering Committee (TSC) and has made a
```

Reply:
@richardlau there are also "emeritus" TSC members as per the README.md which is why I had it that way.  Not sure if it would be obivous if emeritus TSC would be elligeable which is what I inteded.

Reply:
Thinking about it more I guess the emeritus implies that you are no longer on the TSC wo will merge.